source/1_engine/rex_engine/rex_engine.sharpmake.cs

```diff
@@ -25,9 +26,11 @@ public RexEngine() : base()
     string relative_source_path = Util.PathGetRelative(Path.Combine(Globals.SourceRoot), SourceRootPath);
     GenerationConfigPath = Path.Combine(Globals.Root, "config", relative_source_path, "generation.json");
     MemoryTagsHeaderFile = Path.Combine(SourceRootPath, "include", "rex_engine", "memory", "memory_tags.h");
+    FileSystemMountHeaderFile = Path.Combine(SourceRootPath, "include", "rex_engine", "filesystem", "mount_roots.h");
 
     // create it here so it's added to the project in case it didn't exist yet
-    CreateMemoryTagFile();
+    TouchGenerationFile(MemoryTagsHeaderFile);
+    TouchGenerationFile(FileSystemMountHeaderFile);
```

Comment:
I wonder how many times this would be used as I can only think of 2 things that would be mounted. Content and Config files. Are there different paths you would like to mount?

Reply:
yes, but more on that in the file system PR